docs/source/models.rst

```diff
@@ -570,26 +567,6 @@ Example requests:
 For more information about serializing pandas DataFrames, see
 `pandas.DataFrame.to_json <https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_json.html>`_.
 
-The predict command accepts the same input formats. The format is specified as command line arguments.
```

Comment:
are these changes expected? merge conflict?

Reply:
@akshaya-a @shivp950 These changes are unrelated to AzureML and should be reverted. They refer to other serving capabilities that MLflow provides and will continue to provide.